plugins/wysiwygarea/plugin.js

```diff
@@ -416,7 +416,8 @@
 				CKEDITOR.tools.setTimeout( onDomReady, 0, this, win );
 			}, this );
 
-			this._.docTitle = this.getWindow().getFrame().getAttribute( 'title' ) || '&nbsp;';
+			// In case of lack of the title attribute, use non-breaking space.
+			this._.docTitle = this.getWindow().getFrame().getAttribute( 'title' ) || '\xa0';
```

Comment:
I tried this with '&NonBreakingSpace;' but there is a problem with the initial `&` in the HTML entity representation. So it appears that the representation is the issue, not the 'space' or 'non-breaking space'? If you try it with `\xA0` the test also passes. Could you confirm? Additionally - could you decide whether a casual space is enough or the non-breaking one is needed? After that - please adjust the solution (and the test).

Reply:
I've added a comment.

The regular space can't be used here as it is treated by screen readers as no content, making them read the URL of the `iframe`.